src/main/java/de/mediathekview/mlib/tool/Duration.java

```diff
@@ -17,12 +17,13 @@
  *    You should have received a copy of the GNU General Public License
  *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package mSearch.tool;
+package de.mediathekview.mlib.tool;
 
 import java.text.DecimalFormat;
 import java.util.ArrayList;
 import java.util.Date;
 
+
 public class Duration {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)